linter/helper.go

```diff
@@ -111,13 +111,13 @@ var DirectorPropertyTypes = map[string]DirectorProps{
 }
 
 func isAlphaNumeric(r rune) bool {
-	return (r >= 'a' && r <= 'z') || (r >= 'A' && r <= 'Z') || (r >= '0' && r <= '9') || r == '_'
+	return (r >= 'a' && r <= 'z') || (r >= 'A' && r <= 'Z') || (r >= '0' && r <= '9')
```

Comment:
The word `alphanumeric` usually refers to alphabets and numbers, and not underscores.

This is also the case in Fastly's documentation, for example

> Must start with an alphanumeric character and contain only alphanumeric characters, underscores, and whitespace.

at https://www.fastly.com/documentation/reference/api/acls/acl/ clearly indicates `alphanumeric characters` don't include underscores.